querydsl-sql/src/main/java/com/querydsl/sql/SQLTemplates.java

```diff
@@ -1030,6 +1036,10 @@ protected void setDummyTable(String dummyTable) {
         this.dummyTable = dummyTable;
     }
 
+    public void setForShareSupported(boolean forShareSupported) {
```

Comment:
I'll kick off the review then.
It's better to have these `protected`, since templates should report whether they support forShare or not